<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="toolbarLayout">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Status: 0 todo/0 done</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="search">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="searchType">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Done</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Label</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Importance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Due Date</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reminder Date</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="tasksLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>803</width>
          <height>0</height>
         </size>
        </property>
        <property name="columnCount">
         <number>6</number>
        </property>
        <column>
         <property name="text">
          <string>Done</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>label</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Importance</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Due date</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Reminder Date</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="title"/>
      </item>
      <item>
       <widget class="QComboBox" name="importance">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Important</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Urgent</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Not important but urgent</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Not urgent but important</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dueDate">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="reminderDate">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addTaskButton">
        <property name="text">
         <string>Add task</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
